ui/components/ui/multi-select.tsx

```
@@ -0,0 +1,1066 @@
+                          className={cn('ml-2 h-4 w-4 cursor-pointer', responsiveSettings.compactMode && 'ml-1 h-3 w-3')}
+                          onClick={(event) => {
+                            event.stopPropagation()
+                            clearExtraOptions()
+                          }}
+                        />
+                      </Badge>
+                    )}
+                  </div>
+                  <div className="flex items-center justify-between">
+                    <div
+                      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)